src/components/Link/index.tsx

```diff
@@ -21,6 +21,7 @@ type LinkProps = {
   rel?: AnchorHTMLAttributes<HTMLAnchorElement>['rel']
   className?: string
   href: string
+  onClick?: () => void
```

Comment:
Can we add a comment indicating that this is mainly for react router and should not be explcitiely used ?

Reply:
@chambo-e Yes, it's added!